src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -360,9 +385,14 @@ protected function runMigration($migration, $method)
             }
         };
 
-        $this->getSchemaGrammar($connection)->supportsSchemaTransactions()
-                    ? $connection->transaction($callback)
-                    : $callback();
+        try {
+            $this->getSchemaGrammar($connection)->supportsSchemaTransactions()
+                ? $connection->transaction($callback)
+                : $callback();
+        } catch (QueryException $e) {
```

Comment:
You'll want to catch any exception here. :)

Reply:
Yes you're right ! Updated. Thx :)